lib/xnvme_be_cbi_async_emu.c

```diff
@@ -46,16 +46,15 @@ struct xnvme_queue_emu {
 XNVME_STATIC_ASSERT(sizeof(struct xnvme_queue_emu) == XNVME_BE_QUEUE_STATE_NBYTES,
 		    "Incorrect size")
 
-int
-_emu_qp_term(struct _emu_qp *qp)
+int static qpair_term(struct qpair *qp)
```

Comment:
"static int" instead of "int static" and on the previous line

Reply:
Thanks! Fixed.